test/types/records/lydia/recordOffset-stringField.linux32.good

```diff
@@ -1,2 +1,2 @@
 0
-28
+36
```

Comment:
This looks like it got pulled into this PR accidentally.

Reply:
Oops, I must have branched from that PR and not from master, thanks for catching that! I need to start inspecting my branches more closely before opening PR's.

Reply:
@mcdobe100: Here's a prompt setting that, if put in your .bashrc/.bash_login, will have your prompt show which git branch you're currently on when you're within a git repo.  I find it invaluable:

```bash
PS1='\[\e[0;34m\]$host$(__git_ps1 $@)$ \[\e[1;34m\]'
```

If you've got a prompt you already like, you can probably just put the `$(__git_ps1 $@)` part into it somewhere if you don't like the other choices here.